common/data_refinery_common/models/gathered_accession.py

```diff
@@ -0,0 +1,84 @@
+from datetime import datetime
+
+from django.db import models
+from django.utils import timezone
+
+
+class GatheredAccession(models.Model):
+    """Gathered accession model."""
+
+    class Meta:
+        db_table = "gathered_accessions"
+
+    code = models.TextField(unique=True)
+    created_at = models.DateTimeField(auto_now_add=True)
+    last_modified_at = models.DateTimeField(auto_now=True)
+    organism = models.TextField()
+    published_date = models.DateTimeField()
+    sample_count = models.PositiveIntegerField(default=0)
+    source = models.TextField()
+    technology = models.TextField()
+
```

Comment:
This occurred to me today but, I think for now we shouldn't delete the accessions, and also add a relationship on `GatheredAccession` that points to the `SurveyorJob` that we create via the automatic_survey command. I think this would help if we end up Surveying things that we did not intend to.

Reply:
Extracted to https://github.com/AlexsLemonade/refinebio/issues/3074